vnext/proposals/active/mouse-reconcile-desktop.md

```diff
@@ -0,0 +1,107 @@
+- macOS has added 'hovered' and marked their View component implementation of `onMouseEnter/Leave` as deprecated [issue TBD]()
+
+### `hovered` example
+
+``` js
+<Pressable style={({ hovered, pressed, focused }) => {
+          console.log(focused);
+          let backgroundColor = 'white';
+          if (hovered) {
+            backgroundColor = 'lightgray';
+          }
+          if (pressed) {
+            backgroundColor = 'lightblue';
+          }
+          return {
+            padding: 10,
+            margin: 10,
+            borderWidth: 1,
+            borderColor: focused ? 'red' : null,
+            backgroundColor,
+            outlineWidth: 0
+          };
+        }}
+      >
+      <Text>Nested pressables</Text>
+    </Pressable>
+  </Pressable>
+</View>
+```
+
+## Outstanding issues
+There are some outstanding issues that need to be address specifically for desktop scenarios when it comes to fully implementing mouse hover and leave behavior. They are as follows:
+
+- Prevent mouse input from being 'eaten' in ScrollView scenarios [issue #5867](https://github.com/microsoft/react-native-windows/issues/5867)
+- Mouse leave event not firing correctly in Scrollable FlatList scenarios [issue #4614](https://github.com/microsoft/react-native-windows/issues/4614)
+- The hover events/indication for overlapping content prioritizes items "closer" in z-order [issue #5901](https://github.com/microsoft/react-native-windows/issues/5901)
+- Remove the tapped event handler on the `View` component [issue #2349](https://github.com/microsoft/react-native-windows/issues/2349)
+
+## onPointerXX callbacks
+The following callbacks are available on `View` component (and get passed through to Pressable) in Windows and macOS to cover the most common use cases where mouse handling would occur.
+
+The below will need to be implemented for macOS as well [issue TBD]()
+
+| API | Args | Returns | Description |
+|:---:|:----:|:-------:|----|
+| onPointerOver | IPointerEvent | void | Fires when a pointing device is moved within the hit test boundaries of a element. |
+| onPointerLeave | IPointerEvent | void | Fires when a pointing device is moved out of the hit test boundaries of an element and all of its children. |
+
+### Example
+
+``` js
+<View onPointerOver={this._onPointerOver} />
+
+private onPointerOver_ = (event: IPointerEvent) => {
+  this.setState({ lastKeyDown: event.nativeEvent.key });
+};
+```
+
+### Callbacks NOT being implemented
+The following are outstanding events that we are *not* looking to implement at this time. But should be represented as possible callbacks that can be revisited later.
+
+| API | Args | Returns | Description |
+|:---:|:----:|:-------:|----|
+| onPointerEnter | IPointerEvent | void | Fires when a pointing device is moved into the hit test boundaries of an element, including its children.|
+| onPointerDown | IPointerEvent | void | Fires when a pointing device's button (or buttons) state is non-negative. |
```

Comment:
As of change #4878, there is no mechanism for apps to get mouse events for right-click. This was a breaking change that we're having to hack an override to in Garrison.

onPointerDown/onPointerUp, assuming they fire for all mouse buttons, would provide us a solution so I'd like to see it prioritized.

Reply:
Thanks! I'll revise

Reply:
@kikisaints Adding to that, onPointerDown/onPointerUp are useful for adding "pressed" visual states for when a button is held down (rather than just hovered).

As far as performing actions based on a press, this spec is missing something along the lines of web's click/auxclick/contextmenu events, either for Touchable or for View directly.

We had been using touchableHandlePress to fire our onPress and onContextMenu callbacks. With #4878, touchableHandlePress is only called for left-click, so we can't fire onContextMenu anymore.

We could potentially emulate using onPointerDown/onPointerUp once it exists, but it would be better if the logic of determining whether a "right-click" happened (i.e. press and release on an element without cancellation) was done by the framework. I'd be perfectly happy if this was implemented as "touchableHandleAuxPress", but you might also want it as part of the View callbacks.